deepmd/entrypoints/test.py

```diff
@@ -39,6 +40,8 @@ def test(
         path where model is stored
     system : str
         system directory
+    datafile: str
```

Comment:
Numpydoc format: 
> The colon must be preceded by a space

```suggestion
    datafile : str
```

See https://numpydoc.readthedocs.io/en/latest/format.html